extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.schema.json

```
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Feature collection",
+    "type": "object",
+    "description": "A feature collection designated by a feature class and property values stored in arrays.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "featureClass": {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
```

This is no longer an index, it's a string.

```suggestion
            "type": "string",
```